### PR TITLE
preparation for RN,

### PR DESCRIPTION
**Pull request by tosirovect2**
preparation for RN,
O365 groups, OD and Teams articles improved.

### CODE REVIEW

File: get-to-know-security-manager/microsoft-teams-screen.md [hunk 1]
```diff
@@ -13,18 +13,42 @@ To start exploring features and reports, navigate to the Microsoft Teams screen
 
 Available management actions for Microsoft Teams are contextually related to the selected object, which means users can click on or hover over something to get a contextual menu that offers actions related to the selected item.
 
-## Available actions
+## All Teams report
+### All Teams Tab
+General overview of the Teams on your tenant. The report will show you the team's Display Name, privacy level, site URL, number of: channels, team members, owners, guests and the date when the team was created.
```
Comment (imesaric) [inline get-to-know-security-manager/microsoft-teams-screen.md]: možda bi počeo rečenicu sa 'Gives a general overview..'
zarez ispred 'and the date when...'

Kopiram iz grammarlya:
Gives a general overview of the Teams on your tenant. The report will show you the team's Display Name, privacy level, site URL, number of channels, team members, owners and guests, and the date when the team was created.

File: get-to-know-security-manager/microsoft-teams-screen.md [hunk 2]
```diff
@@ -13,18 +13,42 @@ To start exploring features and reports, navigate to the Microsoft Teams screen
...
 * **Clone Team** – – Create a team using an existing team as a template.
-* **Update Team Settings** – Update the properties of the selected team.
-* **Remove Team** – Remove the selected team from SysKit Security Manager.
-* **Delete Team** – Delete the selected team from your tenant.
+* **Update  Settings** – Update the properties of the selected team.
+* **Remove** – Remove the selected team from SysKit Security Manager.
+* **Delete** – Delete the selected team from your tenant.
+
+### Global Settings tab
+On this report, you can see the set values for all the Team settings. They are grouped in 4 sections: Messaging, External Access, Guest Access, Teams Settings
```
Comment (imesaric) [inline get-to-know-security-manager/microsoft-teams-screen.md]: '.. Guest Access, **and** Teams Settings.' - dodati **and** i točku na kraju

File: get-to-know-security-manager/microsoft-teams-screen.md [hunk 3]
```diff
@@ -13,18 +13,42 @@ To start exploring features and reports, navigate to the Microsoft Teams screen
...
+__Global actions:__
 * **Add Owner** – Add owner to an Office 365 group. User availability in the Teams app depends on Team settings and Microsoft user synchronization between Office 365 and Microsoft Teams; this can take up to 48 hours.
 * **Add Member** – Add member to an Office 365 group. User availability in the Teams app depends on Team settings and Microsoft user synchronization between Office 365 and Microsoft Teams; this can take up to 48 hours.
-* **Remove Owner** – Remove the selected owner from the team.
-* **Remove Member** – Remove the selected member from the team.
 * **Go to Group** – Open the associated Office 365 group for a given team.
 * **Go to Site Collection** – Open the associated site collection for a given team.
 
+### Team Members tab
+A detailed overview of a single Microsoft Team, that is, a list of all the team members. The report will show you the users Display name, email, username and role.
```
Comment (imesaric) [inline get-to-know-security-manager/microsoft-teams-screen.md]: The report will show you the **users'** Display name, email, username**,** and role.

File: get-to-know-security-manager/microsoft-teams-screen.md [hunk 4]
```diff
@@ -13,18 +13,42 @@ To start exploring features and reports, navigate to the Microsoft Teams screen
...
+
+__Contextual actions:__
+* **Remove** – Remove the selected user from the team.
+
+### Channels tab
+On this report you can see a list of all the channels in one Team. We provide you with the channels Display name, description, URL and email
```
Comment (imesaric) [inline get-to-know-security-manager/microsoft-teams-screen.md]: On this report**,** you can see a list of all the channels in one Team. We provide you with the channels Display name, description, URL**,** and email**.**

File: get-to-know-security-manager/office-365-groups-screen.md [hunk 5]
```diff
@@ -9,18 +9,40 @@ date: 05/04/2018
 
 To start exploring features and reports, navigate to Office 365 Groups and select your tenant.
 
-Available management actions for Office 365 Groups are contextually related to the selected object, which means users can **click on or hover over something to get a contextual menu** that offers actions related to the selected item.
 
-## Available actions
+You will notice an All Groups overview and Single Group reports. Each report has two tabs. All Groups or Group Members respectively and an individual Audit tab.
+While you can observe and perform drill actions from Snapshots, Live dropdown selection is where other management actions become available.
+Some actions are globally related to Office 365 Groups and others are contextually related to the selected object/s. Which means users can click on a checkbox to fill Actions Toolbar with actions related to the selected item/s.
```
Comment (imesaric) [inline get-to-know-security-manager/office-365-groups-screen.md]: ovaj tu dio bi vidio s Damirom da to malo poenglezi

File: get-to-know-security-manager/office-365-groups-screen.md [hunk 6]
```diff
@@ -9,18 +9,40 @@ date: 05/04/2018
...
+* **Delete Group** – Delete Office 365 Group from your tenant.
+
+
+### Audit tab
+ to do igor
+
+
+## Single group report
+### Groups Members tab
+A detailed overview of a single Office 365 group. The report will show you the users Display name, email, username and role
```
Comment (imesaric) [inline get-to-know-security-manager/office-365-groups-screen.md]: ..**users'** Display name, email, username**,** and role.

File: get-to-know-security-manager/onedrive-screen.md [hunk 7]
```diff
@@ -1,14 +1,41 @@
...
+
+Some actions are globally related to OneDrive's and others are contextually related to the selected object/s. Which means users can click on a checkbox to fill Actions Toolbar with actions related to the selected item/s.
 
 ## Available actions
 
+
+## OneDrive Overview report
+### OneDrive Overview tab
+
+General overview of all the users OneDrive's. On this report you can see a list of users, their corresponding OneDrive URLs, different levels of storage usage, and activity.
```
Comment (imesaric) [inline get-to-know-security-manager/onedrive-screen.md]: zarez iza 'On this report'

**Comment by tosirovect2** [comment]
> Napisao par komentara. Ništa kritično.

par komentara? ubio si me :P 
popravio sam odmah sve, ty :)

**Pull request merged**